src/group_theory/subgroup/basic.lean

```diff
@@ -1964,6 +1978,67 @@ begin
   exact ((f.lift_of_right_inverse f_inv hf).apply_symm_apply _).symm,
 end
 
+/--`mul_hom` on subgroups induced by `mul_hom` of parent group-/
+def img_mul_hom  (H : subgroup G) (f : G →* G₁) :
```

Comment:
`img_mul_hom` what follows are no longer used in this PR, so you might consider splitting them off into a separate PR.